openapi.yml

```diff
@@ -1649,6 +1649,194 @@ paths:
                   version:
                     type: string
           description: ''
+  /osidb/api/v1/flaw_references:
```

Comment:
My local instance has kinda broken Bugzilla-related things, so I am working with partially disabled githooks. Therefore, I cannot guarantee that the `openapi.yml` file is correct, so I rely on CI tests here.

Reply:
Generating the API documentation is based on DRF stuff and should not be blocked by broken Bugzilla integration or failing Bugzilla tests. Just run `make update-schema` and `podman exec -it testrunner tox -e schema`